### PR TITLE
Registrar Inter Account Transfer

### PR DESCRIPTION
**Pull request by simon-says**
### Summary

Documentation to support the manual process of registrar inter-account transfers 

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/fundamentals/setup/manage-domains/move-domain.mdx [hunk 1]
```diff
@@ -15,20 +15,16 @@ You will have to move or transfer domains from one Cloudflare account to another
 * Registered a Cloudflare account with a typo in your email.
 
 :::caution
+If you are using Cloudflare Registrar, you need to submit a manual transfer request.
```
Comment (TownLake) [inline src/content/docs/fundamentals/setup/manage-domains/move-domain.mdx]: If your domain is registered with Cloudflare Registrar, you need to submit a manual request to transfer the domain and its registration to a new account.

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+pcx_content_type: how-to
+title: Transfer a Cloudflare Registrar domain registration between accounts
+---
+
+Cloudflare supports the transfer of domain registrations between Cloudflare accounts when the source and target account both confirm the transfer. The transfer will result in the loss of all configurations and settings for the domain in the source account.
+
+:::caution[Important]
+
+This process only applies to domains which are registered with Cloudflare Registrar. For domains with other Registrar, refer to [Move a domain between Cloudflare accounts
```
Comment (TownLake) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]: nit: for domains with other registrars*

File: src/content/docs/fundamentals/setup/manage-domains/move-domain.mdx (hunk 1 above)
Comment (simon-says) [reply src/content/docs/fundamentals/setup/manage-domains/move-domain.mdx]: Nice - done!

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx (hunk 2 above)
Comment (simon-says) [reply src/content/docs/registrar/account-options/inter-account-transfer.mdx]: Good catch - fixed!

**Review by TownLake — APPROVED** [review]

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+:::
+
+Before proceeding, please be aware of the following:
+
+* WHOIS contact information will be transferred as is.
+* No other configuration will be transferred.
+* After successful transfer, the registration will be transfer locked for 30 days.
+* The target account will become responsible for domain renewals going forward.
+
+## 1. Preparing for the transfer
```
Comment (jason-cf) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
## 1. Prepare for the transfer
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Before you request the transfer, you need to disable DNSSEC for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
+
+## 2. Submitting a transfer request
```
Comment (jason-cf) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
## 2. Submit a transfer request
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Before proceeding, please be aware of the following:
+
+* WHOIS contact information will be transferred as is.
+* No other configuration will be transferred.
+* After successful transfer, the registration will be transfer locked for 30 days.
+* The target account will become responsible for domain renewals going forward.
+
+## 1. Preparing for the transfer
+
+Before you request the transfer, you need to disable DNSSEC for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
```
Comment (jason-cf) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
Before you request the transfer, you need to [disable DNSSEC](/dns/dnssec/#disable-dnssec) for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## 2. Submitting a transfer request
+
+Navigate to the Support and select Contact Cloudflare from the Cloudflare dashboard. Select the category "Registrar" and then "Transfer a domain between Cloudflare accounts". Select the domain(s) in your account that you wish to transfer, enter a Summary and click "Open a Ticket".
```
Comment (jason-cf) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
Navigate to the Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** > **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**. 
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+pcx_content_type: how-to
+title: Transfer a Cloudflare Registrar domain registration between accounts
+---
+
+Cloudflare supports the transfer of domain registrations between Cloudflare accounts when the source and target account both confirm the transfer. The transfer will result in the loss of all configurations and settings for the domain in the source account.
+
+:::caution[Important]
+
+This process only applies to domains which are registered with Cloudflare Registrar. For domains with other registrars, refer to [Move a domain between Cloudflare accounts
+](/fundamentals/setup/manage-domains/move-domain/).
```
Comment (marciocloudflare) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
This process only applies to domains which are registered with Cloudflare Registrar. For domains with other registrars, refer to [Move a domain between Cloudflare accounts](/fundamentals/setup/manage-domains/move-domain/).
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+:::
+
+Before proceeding, please be aware of the following:
+
+* WHOIS contact information will be transferred as is.
+* No other configuration will be transferred.
+* After successful transfer, the registration will be transfer locked for 30 days.
```
Comment (marciocloudflare) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
* After successful transfer, the registration will be transfer-locked for 30 days.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx (hunk 6 above)
Comment (marciocloudflare) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
Go to the Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** and then **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**. 
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+Before you request the transfer, you need to disable DNSSEC for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
+
+## 2. Submitting a transfer request
+
+Navigate to the Support and select Contact Cloudflare from the Cloudflare dashboard. Select the category "Registrar" and then "Transfer a domain between Cloudflare accounts". Select the domain(s) in your account that you wish to transfer, enter a Summary and click "Open a Ticket". 
+
+The form will ask you to confirm that you understand the process and ask you to specify an email address of a new superadmin for the account you intend to transfer the domain(s) to. The new super admin you specify will need to reply to the support ticket to agree to receive the transfer.
+
+Once this has been done, the transfer takes around 3-5 working days to complete.
```
Comment (marciocloudflare) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]:
```suggestion
Once this has been done, the transfer takes around three to five working days to complete.
```

File: src/content/docs/registrar/account-options/inter-account-transfer.mdx [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+* After successful transfer, the registration will be transfer-locked for 30 days.
+* The target account will become responsible for domain renewals going forward.
+
+## 1. Prepare for the transfer
+
+Before you request the transfer, you need to [disable DNSSEC](/dns/dnssec/#disable-dnssec) for the domain(s) and ensure the domain(s) are set up and ready in the new dashboard account you intend to transfer them to.
+
+## 2. Submit a transfer request
+
+Navigate to the Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** > **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**.
```
Comment (jason-cf) [inline src/content/docs/registrar/account-options/inter-account-transfer.mdx]: Go to the Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** and then **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**. 
```suggestion
Go to the Support and select **Contact Cloudflare** from the Cloudflare dashboard. Select the category **Registrar** and then **Transfer a domain between Cloudflare accounts**. Select the domain(s) in your account that you wish to transfer, enter a Summary and select **Open a Ticket**. 
```

**Review by jason-cf — APPROVED** [review]

**Pull request merged**